pkg/localmetrics/localmetrics.go

```diff
@@ -188,12 +189,23 @@ func resourceFrom(url *neturl.URL) (resource string) {
 		// If we can't parse, return a general bucket. This includes paths that don't start with
 		// /api or /apis.
 		if r := recover(); r != nil {
-			// TODO(efried): Should we be logging these? I guess if we start to see a lot of them...
+			// TODO(bdematteo): Should we be logging these? I guess if we start to see a lot of them...
```

Comment:
What's the reason we rotate the assignee around inside the code?

Reply:
This is probably my fault. In my previous life, leaving a name on a TODO was required. It didn’t necessarily mean that person was responsible for _fixing_ the issue, but it at least gave readers a hint as to whom to ask about it. I think Ben probably just changed it to his name when he pasted that line in (in a prior commit).

Reply:
Yes, this is a common function that we sadly had a copy/paster, and I initially didn't see the TODO was named. But as I have more context on the implementation in this operator, I thought it would be more meaningful.